README.md

```diff
@@ -1,3 +1,11 @@
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Travis (.org) branch](https://app.travis-ci.com/ga4gh/refget-compliance-suite.svg?branch=master
+)](https://app.travis-ci.com/github/ga4gh/refget-compliance-suite)
```

Comment:
looks like an unnecessary newline was introduced here?

Reply:
removed the new line